eth2/state_processing/src/per_epoch_processing.rs

```diff
@@ -1,7 +1,7 @@
 use errors::EpochProcessingError as Error;
 use tree_hash::TreeHash;
 use types::*;
-use validator_statuses::{TotalBalances, ValidatorStatuses};
+pub use validator_statuses::{TotalBalances, ValidatorStatuses};
```

Comment:
Had to add a pub here in order to use `ValidatorStatuses` in `operation_pool/src/lib.rs`